packages/ecmascript-runtime-client/modern.js

```diff
@@ -0,0 +1,2 @@
+require("core-js/modules/es7.string.pad-start");
+require("core-js/modules/es7.string.pad-end");
```

Comment:
This set of `core-js` polyfills is definitely too small for the current `setMinimumBrowserVersions` constraints. Further consideration will be necessary, but I'm sure we can do (much) better than `legacy.js`.